dagmc_stats.py

```diff
@@ -30,24 +28,107 @@ def get_dagmc_tags(my_core):
     return dagmc_tags
 
 
-def get_entity_ranges(my_core, meshset, entity_types):
+def get_entity_ranges(my_core, meshset, entity_types, dagmc_tags):
     """
     Get a dictionary with MOAB ranges for each of the requested entity types
-
     inputs
     ------
     my_core : a MOAB Core instance
     meshset : a MOAB meshset to query for the ranges of entities
     entity_types : a list of valid pyMOAB types to be retrieved
-
+    dagmc_tags : the MOAB tags created in a file
     outputs
     -------
     entity_ranges : a dictionary with one entry for each entity type that is a Range of handles to that type
     """
 
     entity_ranges = {}
-    for entity_type in entity_types:
+    for entity_type in entity_types:#goes through each entitytype given
         entity_ranges[entity_type] = my_core.get_entities_by_type(meshset, entity_type) 
-
+        if entity_type == 11: #if the type is mbentitytypes
```

Comment:
You should avoid magic numbers.
maybe consider something like a `C++` enum if that exist in Python ?

Reply:
PEP8: in inline comment `#` should be surrounded with space